tests/tests_earth/tests_sensors.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+from astropy import units as u
+from astropy.tests.helper import assert_quantity_allclose
+from poliastro.bodies import Earth
```

Comment:
`isort` is complaining about this line

Reply:
I checked running tox in master to find out if the error was on any recent change, and it seems the warning is still in master.

Reply:
Confirmed it was an old `isort` version